sdk/src/main/java/com/kin/ecosystem/transfer/presenter/AccountInfoPresenter.java

```diff
@@ -0,0 +1,107 @@
+package com.kin.ecosystem.transfer.presenter;
+
+import android.app.Activity;
+import android.os.AsyncTask;
+
+import com.kin.ecosystem.Kin;
+import com.kin.ecosystem.base.BasePresenter;
+import com.kin.ecosystem.common.exception.ClientException;
+import com.kin.ecosystem.transfer.AccountInfoManager;
+import com.kin.ecosystem.transfer.view.IAccountInfoView;
+
+import java.io.File;
+
+public class AccountInfoPresenter extends BasePresenter<IAccountInfoView> implements IAccountInfoPresenter {
+
+    private AccountInfoManager accountInfoManager = new AccountInfoManager();
+    private AccountInfoAsyncTask asyncTask;
+    private File fileDir;
+    private Activity activity;
+
+    public AccountInfoPresenter(File aFile, Activity aActivity) {
```

Comment:
move `AccountInfoManager ` to preseter constactor.
And activity to `AccountInfoManager ` constactor.

Reply:
done